python/feathub/feature_tables/sinks/memory_store_sink.py

```diff
@@ -16,27 +16,24 @@
 from feathub.feature_tables.sinks.sink import Sink
 
 
-class OnlineStoreSink(Sink):
+class MemoryStoreSink(Sink):
```

Comment:
Shall we rename this class to `MemorySink` instead of `MemoryStoreSink`, in correspondence to `RedisSink` and `KafkaSink`?

Reply:
It is not clear whether MemorySink refers to a queue or kv store. So it seems more readable to name it `MemoryStoreSink`.

Redis is typically used as kv store. And other projects such as Apache Bahir name it RedisSink. So it seems better to keep the same name here.